### PR TITLE
[NEXUS-5399] Expose "Item Max Age" on UI / via REST

### PR DESCRIPTION
**Pull request by adreghiciu**
The new resource field is an optional Integer to avoid existent code to unwanted override of item max age with zero. In that case existent default value will be kept.

### CODE REVIEW

**Comment by cstamas** [comment]
+1

**Comment by adreghiciu** [comment]
CI :+1: https://bamboo.zion.sonatype.com/browse/NXO-OSSF19-2

File: nexus/nexus-webapp/src/main/webapp/js/repoServer/resources.js [hunk 1]
```diff
@@ -153,6 +153,7 @@ define('repoServer/resources',['sonatype/all'], function(){
       notFoundCacheTTL : 'This controls how long to cache the fact that a file was not found in the repository.',
       artifactMaxAge : 'This controls how long to cache the artifacts in the repository before rechecking the remote repository. In a release repository, this value should be -1 (infinite) as release artifacts shouldn\'t change.',
       metadataMaxAge : 'This controls how long to cache the metadata in the repository before rechecking the remote repository. Unlike artifact max age, this value should not be infinite or Maven won\'t discover new artifact releases.',
+      itemMaxAge : 'This controls how long to cache the items in the repository before rechecking the remote repository. The value is used for repository items which are nor artifacts nor metadata.',
```
Comment (nabcos) [inline nexus/nexus-webapp/src/main/webapp/js/repoServer/resources.js:156]: shouldn't this be 'neither ... nor'?

**Comment by nabcos** [comment]
+1

File: nexus/nexus-webapp/src/main/webapp/js/repoServer/resources.js (hunk 1 above)
Comment (adreghiciu) [inline nexus/nexus-webapp/src/main/webapp/js/repoServer/resources.js:156]: I think the native english speaking guys should change it

**Pull request merged**